### PR TITLE
Make CI tests not fail

### PR DESCRIPTION
**Pull request by dmunozv04**
Makes tests work again, since the latest ubuntu version doesn't have python3.6 support.

## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.
If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to the project: _Put an `x` in the boxes that apply_

-   [x] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of
them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before
merging your code._

-   [x] I have read the [CONTRIBUTING](https://github.com/Wirecloud/wirecloud/blob/develop/CONTRIBUTING.md) doc
-   [x] I have signed the [CLA](https://fiware.github.io/contribution-requirements/individual-cla.pdf)
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I have added necessary documentation (if appropriate)
-   [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you
did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by aarranz** [comment]
@dmunozv04, tests are still failing. Any progress on this?

**Comment by dmunozv04** [comment]
@aarranz making tests run again has been a bigger task than it looked, since some selenium methods have changed; but I'll have a commit with the fixes soon

File: src/requirements.txt [hunk 1]
```diff
@@ -12,8 +12,8 @@ regex
 markdown
 django-haystack>=2.8.0,<2.9.0
 whoosh>=2.7.2
-pycrypto
-pyScss>=1.3.4,<2.0
+pycryptodome
+git+https://github.com/Kronuz/pyScss@73559d04 # newest PyScss version, not yet released
```
Comment (aarranz) [inline src/requirements.txt]: Why upgrading pyScss?

Comment (dmunozv04) [reply src/requirements.txt]: The version on pypi is incompatible with python 3.11 
https://github.com/Kronuz/pyScss/issues/428

**Comment by aarranz** [comment]
Working  => merged.

**Pull request merged**